sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -160,7 +160,8 @@ private boolean shouldApplyScopeData(final @NotNull CheckIn event, final @NotNul
         scope != null ? scope.withSession((@Nullable Session session) -> {}) : null;
     @Nullable Session session = null;
 
-    if (event != null) {
+    // Feedbacks shouldn't be sampled, and they don't affect sessions
```

Comment:
Do we need to exclude it from beforeSend/EventProcessors and shall we apply the scope data? In web feedbacks I only see tags being applied, so I imagine everything else is redundant?

Reply:
After reading the internal thread, I'm thinking we should rather introduce a new method in SentryClient (`captureFeedback`) that only does the necessary things for the feedback envelope item type, wdyt? Just to make it easier to distinguish

Reply:
agree, but i'll wait a little until [docs are updated](https://github.com/getsentry/sentry/issues/88232#issue-2956830633)